01LessonOne/LessonOne.js

```diff
@@ -2,36 +2,62 @@ module.exports = function() {
 
   this.returnTrue = function () {
     // should return true
-
+	  return true;
+      
   }
 
   this.returnFalse = function() {
     // should return false
+	  return false;
 
   }
 
   this.greaterThan = function (big, small) {
     // should return true if the first argument is greater than the second argument
     // and return false if the first argument is less than the second argument
+	  if (big > small) {
+		  return true;
+	  }
+	  else {
+		  return false;
+	  }
 
   }
 
   this.lessThan = function (small, big) {
     // should return true if the first argument is less than the second argument
     // and return false if the first argument is greater than the second argument
+	  if (small < big) {
```

Comment:
This is similar to the `big > small` issue. The same comment applies to here, this.equalTo, and this.isEven